### ISSUE TITLE
[Bug]: `.run/Generate Demo Baseline Profile.run.xml` is no longer valid

### ISSUE THREAD

**Issue by SimonMarquis** [android/nowinandroid#859]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is there a StackOverflow question about this issue?

- [X] I have searched StackOverflow

### What happened?

There is a [`Generate Demo Baseline Profile`](https://github.com/android/nowinandroid/blob/main/.run/Generate%20Demo%20Baseline%20Profile.run.xml) run configuration in this project that is currently trying to execute this task:

`:benchmark:pixel6Api31atdDemoBenchmarkAndroidTest --rerun --enable-display -Pandroid.testInstrumentationRunnerArguments.androidx.benchmark.enabledRules=BaselineProfile`

Unfortunately, this configuration no longer exists (the module name changed, the device name changed, etc.).
We should either modify it, or remove it.

I would suggest to "update" it to target any "connected" device with `:benchmark:connectedDemoBenchmarkAndroidTest` to allow anyone to try and test the baseline profile generation.
Can anyone confirm what is the actual task executed to regenerate [baseline-prof.txt](app/src/main/baseline-prof.txt)?

Ping @mlykotom, @zsmb13, @keyboardsurfer.
🙏 sorry for the ping, you are the one updating this file according to the [history](https://github.com/android/nowinandroid/commits/main/app/src/main/baseline-prof.txt)

### Relevant logcat output

```shell
Cannot locate tasks that match ':benchmark:pixel6Api31atdDemoBenchmarkAndroidTest' as task 'pixel6Api31atdDemoBenchmarkAndroidTest' not found in project ':benchmarks'. Some candidates are: 'pixel6api31aospDemoBenchmarkAndroidTest'.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by SimonMarquis** [comment]
Alternatively, what are your thoughts about implementing the already documented ([yet still in alpha](https://maven.google.com/web/index.html?q=benchmark#androidx.benchmark:benchmark-baseline-profile-gradle-plugin)) [Baseline Profile Gradle plugin](https://developer.android.com/topic/performance/baselineprofiles/create-baselineprofile#baseline-profile-gradle-plugin)?

**Comment by keyboardsurfer** [comment]
Going with the Gradle plugin is the best path forward. Is this something you'd like to look into @SimonMarquis ?

**Comment by SimonMarquis** [comment]
Yes, I tried to implement it a few weeks/months ago but never got to make it fully compile.
I'll try again this week.
Can you also give us the current steps (devices, tasks, scripts, etc.) you are following to do regular baseline profile updates?

**Comment by keyboardsurfer** [comment]
Thanks for the feedback. You can set it up [following these instructions](https://developer.android.com/topic/performance/baselineprofiles/create-baselineprofile#baseline-profile-gradle-plugin). Feel free to assign it to me for review and comment me in on any issues you face.
